jacob-hawkins/disable-opportunities-86

jacob-hawkins opened the pull request:
# Description
Allowed fields to always be visible. Disabled when no selection is made, as well as when selection is false. Lowered opacity of disabled fields.

## Relevant issue(s)
#86 

## Questions
<!-- Please note any questions or concerns about this PR here, or specific areas you would like reviewers to pay special attention to -->

## Type of change

<!-- Please delete options that are not relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have assigned reviewers to this PR

cfisher36 reviewed (commented):
Looks great! I made some changes to check "No" by default for clarity. Also, I disabled the input for donation coordinator and phone number if when "No" is checked for donations. Let me know if you see any validation issues arising from these changes

jacob-hawkins commented:
Looks good. I added defaultChecked to the pickup subfield just for more clarity. I also changed where a validation error was being placed. Shouldn't even see there error anymore since its default checked but just in case.
